frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -238,8 +238,27 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
       popUpContext.setLoading(true);
       getProperty(id)
         .then(apiProperty => {
+          const propertyData = apiProperty.data;
+          const propertyAddress = propertyData.address;
+          const teunure = propertyData?.tenure !== undefined ? propertyData?.tenure[0] : undefined;
           const property: IProperty = {
-            ...apiProperty.data,
+            ...propertyData,
+            pid: propertyData.pid || '',
+            status: propertyData.status?.id,
+            dataSource: propertyData.dataSource?.id,
+            tenure: teunure?.id,
+            address: {
+              addressTypeId: '',
+              provinceId: propertyAddress?.province?.id || 0,
+              streetAddress1: propertyAddress?.streetAddress1 || '',
+              streetAddress2: propertyAddress?.streetAddress2,
+              streetAddress3: propertyAddress?.streetAddress3,
+              municipality: propertyAddress?.municipality || '',
+              postal: propertyAddress?.postal || '',
+            },
+            areaUnit: propertyData.areaUnit?.id,
+            landArea: propertyData.landArea || 0,
+            landLegalDescription: propertyData.landLegalDescription || '',
```

Comment:
The property information coming back from the backend does not match the models we use on the front end (IProperty). @asanchezr @devinleighsmith am not too familiar with the leaflet code; can you take a look to see if this mapping is sufficient?

Reply:
I think this code should be using API_Property but you could double check what is returned from the `PropertyController.GetProperty(int id)`

Reply:
IProperty was the old one from CITZ pims, I believe the one I created in the frontend `API_Property` matches the backend `PropertyModel` under the Property area

Reply:
The reason why I believe it does not belong with the shared API_* models is that the model coming from the backend is not common and does not match our current standards yet (I think there are some fields that are not matching the DB). Once we get a model on the same level as the new Address/person/org we can match it with an API_Property. That distinction helps with the refactor.

Reply:
Fair points. We do need to do some cleaning on the various models in the backend. I agree we should keep this separate from API_* until refactor happens .

Reply:
Yes, we agreed to do a progressive refactor, so we shouldn't be expecting anything other then person and possibly some lease models to match our new standards. We are explicitly scheduling that work as tech debt, so refactoring property will wait until we create/assign that story.